lib/azure/armrest/storage_account_service.rb

```diff
@@ -16,22 +16,29 @@ class StorageAccountService < ArmrestService
       def initialize(_armrest_configuration, options = {})
         super
         @provider = options[:provider] || 'Microsoft.Storage'
-        set_service_api_version(options, 'storageAccounts')
+        #set_service_api_version(options, 'storageAccounts')
+        @api_version = '2015-05-01-preview' # Must hard code for now
       end
 
       # Return information for the given storage account name for the
       # provided +group+. If no group is specified, it will use the
       # group set in the constructor.
       #
+      # If the +include_keys+ option is set to true, then the keys for that
+      # storage account will be included in the output as well.
+      #
       # Example:
       #
       #   sas.get('portalvhds1234', 'Default-Storage-CentralUS')
       #
-      def get(account_name, group = armrest_configuration.resource_group)
+      def get(account_name, group = armrest_configuration.resource_group, include_keys = false)
         raise ArgumentError, "must specify resource group" unless group
 
         url = build_url(group, account_name)
-        JSON.parse(rest_get(url))
+        results = JSON.parse(rest_get(url))
+        results['properties'].merge!(list_account_keys(account_name, group)) if include_keys
+
+        results
       end
```

Comment:
The list_account_keys method used to return the same information as get, but with the key information as well. But, something changed on Microsoft's side fairly recently, and now it only returns the keys without the rest of the account info.

I'm pretty sure we'll need the keys to get at the metrics data. Keep in mind that it's optional, and false by default.